packages/web-components/fast-components/src/avatar/scenarios/index.html

```diff
@@ -0,0 +1,27 @@
+<template title="Default">
+    <div>
+        <style>
+            /* For example purposes only. App authors needs to define */
+            fast-avatar {
+                --avatar-fill-accent-primary: #cf4073;
+                --avatar-fill-accent-secondary: #0078d4;
+                --avatar-color-foo: hsl(0, 0%, 100%);
+                --avatar-text-ratio: 2.5;
+                --badge-fill-primary: var(--accent-fill-rest);
+            }
+
+            fast-badge {
+                right: 0px;
+                bottom: 0px;
+            }
+        </style>
+        <fast-avatar
+            alt="Annie's profile image"
+            link="https://microsoft.com"
+            name="Carlos Riviera"
```

Comment:
You need 2 more things for the scenarios/component explorer, a config file and to export that config file from the index.ts in that same directory. pasting an example of the anchor version
![Screen Shot 2020-12-15 at 1 33 23 PM](https://user-images.githubusercontent.com/25805778/102275462-4537a780-3eda-11eb-8e4c-e8817b36175a.png)